packages/pds-ember/app/styles/pds/components/external-link/__config.scss

```diff
@@ -0,0 +1,15 @@
+///-------------------------------------------------------------------------///
+/// WARNING: Private Sass module (definitions may change at ANY time)
+///-------------------------------------------------------------------------///
+$_module: "PDS.Components.ExternalLink";
+/* --- [debug] CONFIG: #{$_module} --- */
+@use "../link/__config" as Link;
+
+@mixin apply {
+  /* [debug] #{$_module}@apply */
+  @include Link.apply {
+    .pds--external {
+      @content;
+    }
+  }
+}
```

Comment:
This module isn't currently used, but it's available in the event that we need to apply styling for external links.